src/styles/keyboard.scss

```diff
@@ -0,0 +1,133 @@
+.keyboard {
+  flex-wrap: nowrap;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  max-width: 1200px;
+  padding: 5px 0;
+  background: #fcfbf2;
+  border: 2px solid black;
+  border-radius: 20px;
+  user-select: none;
+  transition: bottom 0.4s;
+  z-index: 250;
+}
+
+.keyboard--hidden {
+  bottom: -100%;
+}
+
+.keyboard__keys {
+  text-align: center;
+}
+
+.keyboard__key {
+  display: inline-flex !important;
+  justify-content: space-around !important;
+  height: 65px;
+  width: 6%;
+  max-width: 90px;
+  margin: 3px;
+  border-radius: 10px;
+  border: 1px solid black;
+  background: rgba(23, 119, 121, 0.7);
+  color: #fff;
```

Comment:
You use `scss` so with that preprocessor will be good to use `css-variables`. Here you can put all your colors into one file and give them a name to each of them. It gives you the ability to keep your colors in one place and you will be agilest if some color needs to be replaced by another color.

It is just a remark, your app is quite small, so it could be done as you did.